src/wallet/rpcwallet.cpp

```diff
@@ -2311,10 +2311,12 @@ static RPCHelpMan listlockunspent()
 static RPCHelpMan settxfee()
 {
     return RPCHelpMan{"settxfee",
-                "\nSet the transaction fee per kB for this wallet. Overrides the global -paytxfee command line parameter.\n"
-                "Can be deactivated by passing 0 as the fee. In that case automatic fee selection will be used by default.\n",
+                "\n(Superseded by the setfeerate RPC in sat/vB, which it is recommended to use instead.)\n"
```

Comment:
Wouldn't `deprecated` instead of `superseded` be more consistent with previous RPC API "changes"?

Reply:
Thanks, will do on next update.

Reply:
done